pandas/core/internals/blocks.py

```diff
@@ -2382,6 +2382,24 @@ def quantile(self, qs, interpolation="linear", axis=0):
         aware = self._holder(res_blk.values.ravel(), dtype=self.dtype)
         return self.make_block_same_class(aware, ndim=res_blk.ndim)
 
+    def _check_ndim(self, values, ndim):
+        """
+        ndim inference and validation.
+
+        This is overriden by the DatetimeTZBlock to check the case of 2D
+        data (values.ndim == 2), which should only be allowed if ndim is
+        also 2.
+        """
+        if ndim is None:
```

Comment:
we recently started always-passing ndim to the constructor, so should never have None here

Reply:
this is backported though, so prob ok (but will need a patch on 1.2)

Reply:
This is also done in the parent method, so can use a general clean-up for 1.2

Reply:
kk can you open an issue so we don't forget to do this